releasenotes.md

```diff
@@ -1,6 +1,17 @@
 GeoIP2 .NET API Release Notes
 =============================
 
+2.4.0-beta1 (2015-09-10)
+------------------------
+
+* Async support was added to the `WebServiceClient`. Each web-service end
+  point now has a corresponding *Async(ip)` method. GitHub #1.
+* Use of RestSharp was replaced by `HttpWebRequest` for synchronous HTTP
+  requests and `HttpClient` for asynchronous requests. Microsoft BCL libraries
+  are used to provide `async`/`await` and `HttpClient` support on .NET 4.0.
+  GitHub #32.
```

Comment:
Issue 32 is "Databasereader Not enough storage not solved #32".

I do not understand the relationship between the 2 sentences- are they talking about .Net 4.0 vs. some other version, or is the 2nd sentence explaining the 1st?